AzFilesHybrid/AzFilesHybrid.psd1

```diff
@@ -12,7 +12,7 @@
     RootModule = 'AzFilesHybrid.psm1'
     
     # Version number of this module.
-    ModuleVersion = '0.2.6.0'
+    ModuleVersion = '0.2.7.0'

```

Comment:
We have to keep the version on azuredeploy.json the same as this module version. Its a convention. https://github.com/Azure-Samples/azure-files-samples/commit/665940bd20dcb7261f29d5a53221ccc9c60ab75c